template_ws/.gitignore

```diff
@@ -1,5 +1,10 @@
-.devcontainer/cache
 .vscode
-build
-install
-log
+
+# ROS2 basic directories
+/build
+/install
+/log
+
+# Gazebo cache
+docker/cache/*
+!docker/cache/.gazebo/.gitkeep
```

Comment:
This part should be:

```
# Gazebo cache
docker/cache/*
!docker/cache/.gazebo
docker/cache/.gazebo/*
!docker/cache/.gazebo/.gitkeep
```

as mentioned here: https://github.com/j3soon/ros2-agv-essentials/pull/13#discussion_r1374202127

The correct way to unignore file is rather ugly... The post below explains why this is required:

See: https://stackoverflow.com/a/5534865